store-implementation/hbase-store/src/main/java/uk/gov/gchq/gaffer/hbasestore/HBaseStore.java

```diff
@@ -180,6 +184,25 @@ protected SchemaOptimiser createSchemaOptimiser() {
     }
 
     @Override
+    public void validateSchemas() {
+        super.validateSchemas();
+        if (!getSchema().getVertexSerialiser().isConsistent()) {
+            throw new SchemaException("Vertex serialiser is inconsistent - store may be unable to handle this.");
```

Comment:
Please could you update the error message to:

"Vertex serialiser is inconsistent. This store requires vertices to be serialised in a consistent way."